src/Command/UpdateCaseDeadlineBooleansCommand.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Command;
+
+use App\Service\CaseManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UpdateCaseDeadlineBooleansCommand extends Command
+{
+    protected static $defaultName = 'tvist1:update-case-deadlines';
+    protected static $defaultDescription = 'Updates case deadline booleans';
+    /**
+     * @var CaseManager
+     */
+    private $caseManager;
+
+    public function __construct(CaseManager $caseManager)
+    {
+        $this->caseManager = $caseManager;
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription(self::$defaultDescription)
```

Comment:
Is this needed? Setting `$defaultDescription` (in line 15) might be enough.

Reply:
I believe this is still needed in 5.2 (https://symfony.com/doc/5.2/console.html#configuring-the-command).

It does however look like it would be enough when we upgrade to 5.3 (https://symfony.com/doc/5.3/console.html#configuring-the-command).